tests/integration/security/authorization_test.go

```diff
@@ -1301,9 +1309,8 @@ func TestAuthorization_Custom(t *testing.T) {
 			if _, _, err := kube.WaitUntilServiceEndpointsAreReady(ctx.Clusters().Default(), extAuthzServiceNamespace.Name(), "ext-authz"); err != nil {
 				ctx.Fatalf("Wait for ext-authz server failed: %v", err)
 			}
-
-			policy := applyYAML("testdata/authz/v1beta1-custom.yaml.tmpl", ns)
-			defer ctx.Config().DeleteYAMLOrFail(t, ns.Name(), policy...)
+			policy := applyYAML("testdata/authz/v1beta1-custom.yaml.tmpl", nil)
```

Comment:
I'm not sure if this is going to work, should the policy for istio-system namespace be put in a separate yaml?

Reply:
I seperate the ingress part in this pr https://github.com/istio/istio/pull/29899, but seems still got the timeout issue.